docs/pages/database-access/guides/postgres-cloudsql.mdx

```diff
@@ -31,43 +31,15 @@ videoBanner: br9LZ3ZXqCk
 Teleport uses [IAM database authentication](https://cloud.google.com/sql/docs/postgres/authentication)
 with Cloud SQL PostgreSQL instances.
 
-If you're [creating](https://console.cloud.google.com/sql/choose-instance-engine)
-a new PostgreSQL instance, make sure to add the `cloudsql.iam_authentication`
-database flag under "Customize your instance / Flags" section:
+(!docs/pages/includes/database-access/cloudsql_enable_iam_auth.mdx type="PostgreSQL" !)
 
-![Enable IAM Authentication](../../../img/database-access/guides/cloudsql/iam-new-instance@2x.png)
-
-To check whether IAM authentication is enabled for an existing Cloud SQL instance,
-look for the flag on the Configuration panel on the instance's Overview page:
-
-![Check IAM Authentication](../../../img/database-access/guides/cloudsql/iam-existing-instance@2x.png)
-
-If it isn't enabled, you can add this flag using the "Edit configuration" dialog
-at the bottom of the Configuration panel. Note, changing this setting may
-require a database instance reboot.
-
-## Step 2/7. Create a service account for the database
+## Step 2/7. Create a service account to access the PostgreSQL instance
```

Comment:
```suggestion
## Step 2/7. Create a service account for a database user
```

likewise with my comment from step 2/7 in mysql guide - if you decide to use my suggestion there